dev-db/mydumper/mydumper-0.10.3.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A high-performance multi-threaded backup (and restore) toolset for MySQL"
+HOMEPAGE="https://github.com/maxbube/mydumper"
+SRC_URI="https://github.com/maxbube/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+COMMON_DEPEND="dev-db/mysql-connector-c:=
+	dev-libs/glib:=
```

Comment:
`:2=`

Reply:
Can we talk about this? This is Gentoo, a rolling distribution. We should only set upper bounds when know for sure it will not work with whatever comes next. But pinning to a slot when this cannot be known yet is against the idea of a rolling distribution. Not?